src/Tickets/QR/Facade.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace TEC\Tickets\QR;
+
+/**
+ * Class Facade
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\QR
+ */
+class Facade {
```

Comment:
I'm not a huge fan of using the word Facade for the actual class name. It can be a facade, but be a name that is more recognizable. Perhaps: `TEC\Tickets\QR\QR`

That way, any operation/interaction is immediately recognizable without the need for a `use` alias.

```php
tribe( QR::class )->get_png_as_string( $data );
```